flutter/lib/src/integrations/sdk_integration.dart

```diff
@@ -0,0 +1,3 @@
+export 'native_sdk_integration.dart'
+    if (dart.library.html) 'web_sdk_integration.dart'
+    if (dart.library.js_interop) 'web_sdk_integration.dart';
```

Comment:
afaik yes because dart:html is not available in wasm but I'll double check

Reply:
` Error: Dart library 'dart:html' is not available on this platform.` on wasm

so if we didnt have the second if statement then it would try to import the native sdk for wasm